index.html

```diff
@@ -148,7 +148,9 @@ <h1>Multi-Post Stories</h1>
   </section>
   <section class="section-2-desktop">
     <div class="container-1">
-      <img src="images/Snapshoot Portfolio@2x.png" />
+      <div>
+        <img src="images/Snapshoot Portfolio@2x.png" />
+      </div>
       <div class="items">
         <h1>Tonic</h1>
         <ul>
```

Comment:
- [ ] 🙏Please adjust your style(margins and padding) to match the Figma design. Doing so will enhance the appearance 🎉of the app's design.
Your design:
![image](https://user-images.githubusercontent.com/63445083/236317525-dcfab901-e66c-47f3-b3da-3381ca68d7e2.png)

Figma design:
![image](https://user-images.githubusercontent.com/63445083/236317810-2ba095ba-e371-487d-a3aa-78e2b6519693.png)